sklearn/cluster/tests/test_hierarchical.py

```diff
@@ -264,6 +273,14 @@ def test_agglomerative_clustering():
     assert_array_equal(clustering.labels_, clustering2.labels_)
 
 
+def test_agglomerative_clustering_mem_mapped():
+    """AgglomerativeClustering must work on mem-mapped dataset.
+    Non-regression test for issue #19875."""
```

Comment:
Nitpick: [PEP 257 style for multi-line docstring](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings):

```suggestion
    """AgglomerativeClustering must work on mem-mapped dataset.
    
    Non-regression test for issue #19875.
    """
```